src/vulnpy/trigger/nosqli.py

```diff
@@ -0,0 +1,52 @@
+from pymongo import MongoClient
+import os
+
+
+def mongo_db_exists():
+    client = MongoClient(serverSelectionTimeoutMS=20)
+    db = client.pymongo_test
+
+    try:
+        db.posts.count_documents({})
+    except Exception as e:  # pragma: no cover
+        print("EXCEPTION", e)
```

Comment:
can you remove this print and handle any printing / logging in the `else` clause of `if mongo_db_exists()`?